arch/arm/soc/st_stm32/stm32f0/soc.h

```diff
@@ -41,6 +41,7 @@
 #include <stm32f0xx_ll_bus.h>
 #include <stm32f0xx_ll_rcc.h>
 #include <stm32f0xx_ll_system.h>
+#include <stm32f0xx_ll_spi.h>
```

Comment:
<stm32f0xx_ll_spi.h> should be garded by #ifdef CONFIG_SPI_STM32 and placed outside of  CONFIG_CLOCK_CONTROL_STM32_CUBE.

Reply:
Yup, thanks for spotting it